.github/workflows/colin-api-ci.yml

```diff
@@ -66,6 +66,10 @@ jobs:
 
     runs-on: ubuntu-24.04
 
+    strategy:
+      matrix:
+        python-version: [3.8]
+
```

Comment:
There were quite a few scripts missing this. (I think only 1 script had this, actually.)